packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1883,19 +1883,21 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     newRoute.install(_currentOverlayEntry);
     _history[index] = newRoute;
     newRoute.didPush().whenCompleteOrCancel(() {
-      // The old route's exit is not animated. We're assuming that the
-      // new route completely obscures the old one.
       if (mounted) {
+        // The old route's exit is not animated. We're assuming that the
+        // new route completely obscures the old one.
```

Comment:
can you revert this comment to move it to its original place?

Reply:
Pushed commit moving this.